x-pack/plugins/lens/public/xy_visualization/annotations_helpers.tsx

```diff
@@ -10,7 +10,11 @@ import React from 'react';
 import { EuiFlexGroup, EuiIcon, EuiIconProps, EuiText } from '@elastic/eui';
```

Comment:
this whole file can be removed, it's only used for renderer and it was already moved to the new plugin. 
Same for `x-pack/plugins/lens/public/xy_visualization/annotations/expression.tsx` (there are no changes so I cannot put a comment there)

Reply:
Thanks, I'll check all the files at Lens one more time. I was doing the before for reference_lines and data layers, and all the xy)

Reply:
Addressed.